src/api-reference/request/v4.endpoints.workflow-resources.markdown

```diff
@@ -0,0 +1,174 @@
+---
+title: Travel Request v4 - Workflow Resources
+layout: reference
+---
+
+{% include prerelease.html %}
+
+* [Workflow Resources](#workflow-resource)
+  * [Move an existing Request in the approval workflow](#Move-existing-request-in-approval-workflow)
+
+## HTTP Status Codes
+
+To learn more about response HTTP status codes for this API see [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html).
+
+# <a name="workflow-resource"></a>Workflow Resources
+
+Manage workflow transitions for a Request.
+
+## <a name="Move-existing-request-in-approval-workflow"></a>Move an existing Request in the approval workflow
+
+### Scopes
+
+`travelrequest.write` - Refer to [Scope Usage](./v4.get-started.html#scope-usage) for full details.
+
+### HTTP Request
+
+#### Parameters
+
+Name|Parameter Type|Data Type|Description
+---|---|---|---
+requestUuid|path|string|**Required** The unique identifier of the Request
+action|path|string|**Required** The state transition to be executed (`submit`, `approve`, `recall`, `sendback`, `cancel`, `close` or `reopen`)
+comment|path|string|Only works with when the workflow action is ‘sendback’. This comment is visible wherever Request comments are available to the employee, approver, and/or Request administrator
```

Comment:
comment, userId, and companyID are listed as path parameters, but the template and example don't show where they go.

Reply:
@sarraloew this is how it has always been documented in previous version, and it seems that we are systematically putting this value in this parameter Type column > is this column mandatory and what added value is it bringing to our API consummers?

@laurentlhermine  as per above comment to sarra we have always set the valeur path to the Parameter Type > what do you think about this value / column? do you think it should be modified or removed?